cypress/e2e/tests/migration/applicationinventory/assessment/questionnaire_features.test.ts

```diff
@@ -92,11 +89,47 @@ describe(["@tier3"], "Tests related to questionnaire features", () => {
         clickByText(button, "Cancel");
         click(confirmButton);
         Assessment.verifyAssessmentTakeButtonEnabled();
-        appConditionalQuestion.delete();
+    });
+
+    it("1) Test auto answer feature of questionnaires; 2) Save assessment", function () {
+        //Automates Polarion MTA-388: Auto answer
+        Application.open();
+        cy.wait(2 * SEC);
+        clickItemInKebabMenu(testApplication.name, "Assess");
+        Assessment.take_questionnaire(cloudReadinessQuestionnaire);
+        Assessment.selectStakeholders(stakeholderList);
+        clickJs(nextButton);
+        cy.wait(SEC);
+
+        // Automates Polarion MTA-
+        cy.get("div.pf-v5-l-split__item")
+            .contains("What is the main technology in your application?")
+            .closest(questionBlock)
+            .within(() => {
+                cy.get("div.pf-v5-c-radio")
+                    .contains("Quarkus")
+                    .closest("div.pf-v5-c-radio")
+                    .within(() => {
+                        cy.get('*[class^="pf-v5-c-radio__input"]').invoke("is", ":checked");
```

Comment:
There are too many selectors, I am afraid any change in UI will impact this test. 
Please see if any of the selector is common among test and move them to views so that less impact on tests.